src/components/Tool/NavToolList/NavToolList.tsx

```diff
@@ -1,61 +1,153 @@
+      totalCount: 0,
+    },
+    [ToolName.FixMailingAddresses]: data?.[ToolName.FixMailingAddresses] ?? {
+      totalCount: 0,
+    },
+    [ToolName.FixSendNewsletter]: data?.[ToolName.FixSendNewsletter] ?? {
+      totalCount: 0,
+    },
+    [ToolName.FixEmailAddresses]: data?.[ToolName.FixEmailAddresses] ?? {
+      totalCount: 0,
+    },
+    [ToolName.FixPhoneNumbers]: data?.[ToolName.FixPhoneNumbers] ?? {
+      totalCount: 0,
+    },
+    [ToolName.MergeContacts]: data?.[ToolName.MergeContacts] ?? {
+      totalCount: 0,
+    },
+    [ToolName.MergePeople]: data?.[ToolName.MergePeople] ?? { totalCount: 0 },
+  };
 
   return (
-    <Box component="div" className={classes.list} data-testid="ToolNavList">
-      <List>
-        {ToolsList.map((group) => (
-          <Fragment key={group.groupName}>
-            <ListItem className={classes.li}>
-              <ListItemIcon>
-                <Icon path={group.groupIcon} size={1} />
-              </ListItemIcon>
-              <ListItemText primary={group.groupName} />
-            </ListItem>
-            {group.items.map((tool) => (
-              <Item
-                key={tool.id}
-                url={tool.url}
-                title={tool.tool}
-                isSelected={selectedId === tool.id}
-              />
-            ))}
-          </Fragment>
-        ))}
-      </List>
+    <Box data-testid="ToolNavList">
+      <div className={classes.root}>
+        <Slide in={isOpen} direction="right" mountOnEnter unmountOnExit>
+          <Box>
+            <FilterHeader>
+              <Box
+                display="flex"
+                justifyContent="space-between"
+                alignItems="center"
+              >
+                <Typography variant="h6">{t('Tools')}</Typography>
+                <IconButton
+                  data-testid="ToolNavToggle"
+                  onClick={() => toggle(!isOpen)}
+                >
+                  <Close titleAccess={t('Close')} />
+                </IconButton>
+              </Box>
+            </FilterHeader>
+            <FilterList dense>
+              {ToolsList.map((group) => (
+                <Fragment key={group.groupName}>
+                  <ListItem
+                    data-testid="ToolNavListItem"
+                    className={classes.li}
+                  >
+                    <ListItemIcon>
+                      <Icon path={group.groupIcon} size={1} />
+                    </ListItemIcon>
+                    <ListItemText primary={t(group.groupName)} />
+                  </ListItem>
+                  {group.items.map((tool) => {
+                    return (
+                      <Item
+                        key={tool.id}
+                        url={tool.url}
+                        title={tool.tool}
+                        isSelected={selectedId === tool.id}
+                        loading={loading}
+                        needsAttention={
```

Comment:
I don't think a separate `needsAttention` prop is needed. In the `Item` component, can't we just skip render the count if `totalCount` is 0?